File: LeetCode Solutions/Problems/2148. Count Elements With Strictly Smaller and Greater Elements.cpp
class Solution {
public:
    int countElements(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        int minimum = nums[0];
        int maximum = nums[nums.size() - 1];
        int count = 0;
        for (int i=0; i<nums.size(); i++){
            if (nums[i] > minimum && nums[i] < maximum){
                count += 1;
            }
        }
        return count;
    }
};

File: LeetCode Solutions/Problems/2148. Count Elements With Strictly Smaller and Greater Elements.py
class Solution:
    def countElements(self, nums: List[int]) -> int:
        minimum = min(nums)
        maximum = max(nums)
        count = 0
        for i in range(len(nums)):
            if nums[i] > minimum and nums[i] < maximum:
                count += 1
        return count